rs/nns/integration_tests/src/update_canister_settings.rs

```diff
@@ -19,16 +19,18 @@ use ic_nns_test_utils::{
     },
 };
 
-#[test]
-fn test_update_canister_settings() {
+fn test_update_canister_settings(canister_id: CanisterId, controller_canister_id: CanisterId) {
```

Comment:
Add role to the name canister_id. E.g. patient_canister_id.

Reply:
Done.